### PR TITLE
now using Yup() for email validation for org invite

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

Bug fix, feature, docs update, ...

## What is the current behavior?

Now accepts any domain for email invite

### CODE REVIEW

File: studio/components/interfaces/Organization/InviteMemberModal.tsx [hunk 1]
```diff
@@ -74,6 +75,10 @@ function InviteMemberModal({ organization, members = [], user }: any) {
     setEmailAddress(e.target.value)
   }
 
+  let schema = object({
```
Comment (joshenlim) [inline studio/components/interfaces/Organization/InviteMemberModal.tsx:78]: Small one - const

**Review by joshenlim — APPROVED** [review]
Just small change

File: studio/components/interfaces/Organization/InviteMemberModal.tsx (hunk 1 above)
Comment (MildTomato) [reply studio/components/interfaces/Organization/InviteMemberModal.tsx:78]: ah

Comment (MildTomato) [reply studio/components/interfaces/Organization/InviteMemberModal.tsx]: thanks, my bad

**Pull request merged**